apps/system/test/unit/wifi_test.js

```diff
@@ -625,27 +542,14 @@ suite('WiFi > ', function() {
       Wifi.wifiEnabled = false;
       navigator.battery = { charging: false };
 
-      var isWifiPowerSavingMode = false;
-      var isSetPowerSavingModeCalled = false;
-      var fakeDOMRequest = {
-        onsuccess: function() {},
-        onerror: function() {}
-      };
       var stubMozWifiManager = this.sinon.stub(navigator,
```

Comment:
Again, remove this redundant stub totally and
https://github.com/tefn3849/gaia/blob/wifi_1127226/apps/system/test/unit/wifi_test.js#L323-L324